recipes-containers/dobby/dobby.bb

```diff
@@ -0,0 +1,51 @@
+SUMMARY = "Dobby Container Manager"
+DESCRIPTION = "Dobby “Docker based Thingy” is a tool for managing and running OCI containers using crun"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=c466d4ab8a68655eb1edf0bf8c1a8fb8"
+
+SRC_URI = "git://github.com/rdkcentral/Dobby"
+
+#2020-12-14
+SRCREV = "2db374a248e1e011a13d7ba643901f7da6972710"
+DEPENDS_append = " libnl dbus jsoncpp python3 ctemplate boost yajl systemd"
+RDEPENDS_${PN} = "crun (>= 0.14.1)"
+
+S = "${WORKDIR}/git"
+
+# Always build the debug version for now
+EXTRA_OECMAKE = "-DCMAKE_BUILD_TYPE=Debug "
+
+# Sky XiOne build
+EXTRA_OECMAKE_append_skyxione = "-DLEGACY_COMPONENTS=ON -DRDK_PLATFORM=XI1 "
+
+# Sky Llama build
+EXTRA_OECMAKE_append_llama = "-DLEGACY_COMPONENTS=ON -DRDK_PLATFORM=LLAMA "
+
+# Comcast Xi6 build
+EXTRA_OECMAKE_append_arrisxi6 = "-DLEGACY_COMPONENTS=OFF -DRDK_PLATFORM=XI6 "
```

Comment:
Could you please make it a PACKAGECONFIG?

Reply:
done